db/main.go

```diff
@@ -9,15 +10,36 @@ import (
 
 var (
 	log = logging.Logger("router")
+	DB  *gorm.DB
 )
 
+type DeltaDM struct {
+	DB         *gorm.DB
+	DryRunMode bool
+	Config     config.DeltaConfig
+}
+
+func NewDeltaDMDm() *DeltaDM {
```

Comment:
Replace the names, i.e `DeltaDM` -> `SpdTenantSvc` 
`NewDeltaDm` -> `NewSpdTenantSvc`